alvr/session/build.rs

```diff
@@ -4,7 +4,7 @@ use std::{env, fmt::Write, fs, path::PathBuf};
 fn main() {
     let openvr_driver_header_string =
         fs::read_to_string(alvr_filesystem::workspace_dir().join("openvr/headers/openvr_driver.h"))
-            .unwrap();
+            .expect("Missing openvr header files, did you clone the submodule?\n");
```

Comment:
Very minor, but I don't think you have to put a \n in the expect message

Reply:
Intentional, the line gets very long if the error is appended to the expect in the same line

Reply:
Alright